ProjectSettings/ProjectSettings.asset

```diff
@@ -760,7 +760,7 @@ PlayerSettings:
     m_VersionCode: 1
     m_VersionName: 
   apiCompatibilityLevel: 6
-  activeInputHandler: 0
+  activeInputHandler: 1
   cloudProjectId: 
```

Comment:
I think it would be helpful to support both for now. The ideal is that we just use the new input system, but supporting both means that anyone can contribute even without knowing the new system and someone else can convert it later.

Reply:
I have reverted the project settings to support BOTH InputSystems.